.jest/setup.js

```diff
@@ -1 +1,9 @@
 import '@testing-library/jest-dom/extend-expect'
+import path from 'path'
+import { unflatten } from 'flat'
+import { getHashiConfig } from '../config'
+
+const env = process.env.HASHI_ENV || 'development'
+const envConfigPath = path.join(process.cwd(), 'config', `${env}.json`)
+
+global.__config = unflatten(getHashiConfig(envConfigPath))
```

Comment:
I need to access the beta product data from config. Lmk if this is the right approach for jest set-up.

Reply:
Sweetness, nice!